config/routes.rb

```diff
@@ -1,6 +1,6 @@
 C2::Application.routes.draw do
   ActiveAdmin.routes(self)
-  root :to => 'home#index'
+  root to: "home#index"
```

Comment:
If you're going to change that one, why not change them all and get it done with?